### ISSUE TITLE
FSE beta: Remove the keep/replace content modal when switching themes

### ISSUE THREAD

**Issue by vindl** [Automattic/wp-calypso#56872]
### Details

We should remove the keep/replace content modal when switching themes for FSE Beta sites:

<img width="755" alt="132927910-e9ca4e19-eb87-43d9-98ac-42666cfd3853" src="https://user-images.githubusercontent.com/1182160/136422333-48f4c014-7f6f-4f3e-8c27-75c40371328a.png">

For universal themes, should they always skip this modal, or only for FSE Beta sites? Needs further testing

### Related

https://github.com/Automattic/wp-calypso/issues/54462

**Comment by creativecoder** [comment]
> For universal themes, should they always skip this modal, or only for FSE Beta sites? Needs further testing

Here's the results of some testing I did today to try and determine the current effects of "Switch theme, preserving my homepage content" and "Replace my homepage content with ..." actions when switching to a universal theme.

When switching to a Universal theme (Quadrat, Geologist, etc) from another Universal theme OR a Classic theme...

**Switch theme, preserving my homepage content.**

Keeps my current home page content, as expected.

**Replace my homepage content...**

If I have previously chosen this option to replace my homepage content when switching to a classic theme, this option replaces that homepage content from the classic theme. However, I don't get the content from the new Universal theme, it's reset to the content that was there before I replaced the content with the Classic theme 😕 .

For example, if I switch to Bingley and replace my content...

<img width="300" alt="image" src="https://user-images.githubusercontent.com/1699996/137809069-9b87a228-f102-450f-bb63-01fd6455a909.png">

Then customize my homepage, I see this as the homepage.

<img width="300" alt="image" src="https://user-images.githubusercontent.com/1699996/137809194-fe74eeb3-6d1c-4cc1-b265-dbff9dd41e26.png">

If I then switch to Quadrat, and chose to replace my content...

<img width="300" alt="image" src="https://user-images.githubusercontent.com/1699996/137809293-6a6dece6-d749-4f6f-996a-8239b9342a3b.png">

I see... Mayland homepage content, I think because Mayland (Blocks) was the first theme I set up when I made the site (gregtest1fse.wordpress.com)

<img width="300" alt="image" src="https://user-images.githubusercontent.com/1699996/137809429-9720b892-544c-44c5-8ade-5492317ce4a6.png">

If I haven't previously replaced my homepage content, nothing changes and my homepage stays the same (Mayland content, in this case)

---

So I'm not sure if we need to remove this option, or modify it to behave differently for Universal themes.

@Automattic/theam @ianstewart for ideas and suggestions.
